### ISSUE TITLE
At times the rangefinder sticks on

### ISSUE THREAD

**Issue by Symon-S** [manuelVo/foundryvtt-rangefinder#2]
Intermittently when scrolling between windows in a multi monitor setup the rangefinder sticks on. This happens without it being used beforehand. I will monitor for bad code next time it happens and post results at that time. The way to clear it when it gets stuck is to click on the token it originates from, press control, click the token again, and press control again.

**Comment by manuelVo** [comment]
Do you press down the button that is configured to be used for showing the rangefinder while switching the windows when this happens?

**Comment by Symon-S** [comment]
No, just move the mouse across to another monitor to click on say monster stats. Then when I return to the foundry window/monitor it is stuck engaged in the rangefinder as if I had pressed the assigned key.

**Comment by manuelVo** [comment]
Hmm I was able to reproduce a similar (but possibly related) bug. But in quick tests I've done I couldn't reproduce it the way you described. If you can provide any additional information to how or why this happens that might help me track down the bug. When does the rangefinder appear exactly? When the focus goes to the other window (that you popped out using the pop out module, I assume?), when you click the the monster stat, when the focus goes back to the main foundry window? Is there a way to somewhat reliably trigger the bug (even if that only works like one time in three or four attempts that would be a start)? Or any other information that might reveal *something* about the nature of the bug.

**Comment by Symon-S** [comment]
I will track when it happens and see if it pops an error code. But it happens when just switching to a AoN in a new browser window. Could be happening with pop outs too, but I usually use AoN for monster stats. When I get it to duplicate, I will see if I have any relevent data to post. I will also monitor the exact moment of occurence and what I was doing. And see if I can make a video of it before toggling it off when it does.

**Comment by Symon-S** [comment]
Seems that it's just a multi-screen issue as it will continue to function if copy/pasting addresses and stuff, might just be that I am not switching focus between the windows as this is being done, or there is a small delay between recognizing the switch in focus while multitasking.
